app/browser/api/ledger.js

```diff
@@ -1875,6 +1960,19 @@ const initialize = (state, paymentsEnabled) => {
     }
   }
 
+  if (!ledgerState.getInfoProp(state, 'referralTimestamp')) {
```

Comment:
shouldn't this check if `downloadId` exists instead of `referralTimestamp`? IIUC, `referralTimestamp` is only set when the activity check is called, after 30 days, whereas `downloadId` is set as soon as the referral is initialized.

Reply:
you are correct, fixed